SimPEG/Regularization.py

```diff
@@ -1384,6 +1510,18 @@ def _mirror_eps_q_to_derivs(self, change):
             if isinstance(objfct, SparseDeriv):
                 objfct.epsilon = change['value']
 
+    @properties.observer('space')
+    def _mirror_space_to_objfcts(self, change):
+        for objfct in self.objfcts:
+            objfct.space = change['value']
+
+def coterminal(theta):
```

Comment:
cool, I think this has more general use, could we put it in coordutils.py and so we use this as `Utils.coterminal()`